content/en/monitors/types/process.md

```diff
@@ -19,21 +19,44 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended to:
+
+- Ensure that enough instances of a non-containerized long-lived process are running.
+- Flag when a specific process is running.
+
+**Note**: Only long-lived processes are picked up by the Agent. Monitors on processes that live for less than 20 seconds may be flaky.
 
 ## Monitor creation
 
-To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
+To create a [Live Process Monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
 
 ### Select processes
 
-Search for a process to monitor using a space separated list of strings. The search performs a partial match, fuzzy search across all processes on your infrastructure. Search operators `AND`, `OR`, and `NOT` are supported. See the [Live Process Monitoring][3] page for details. Additionally, filter by tags to refine your monitor to a specific scope.
-
-Matching processes and counts are displayed below the search:
+You can use either tags or a fuzzy search to filter across all processes in your infrastructure. Matching processes and counts are displayed below the search:
 
 {{< img src="monitors/monitor_types/process/select_processes.png" alt="Select processes" style="width:90%;">}}
 
-After defining your search, a graph is displayed above the search inputs with an approximation of the total number of processes found. For more granular data, visit your [Live Process Page][4].
+After defining your search, a graph is displayed above the search inputs with an approximation of the total number of processes found. It's recommended to keep your monitor scoped to a few thousands processes. Use additional tags to narrow the search down or consider splitting a monitor into multiple ones if needed. For more granular data, visit your [Live Process Page][4].
```

Comment:
Nit: It's -> It is
```suggestion
After defining your search, a graph is displayed above the search inputs with an approximation of the total number of processes found. It is recommended to keep your monitor scoped to a few thousands processes. Use additional tags to narrow the search down or consider splitting a monitor into multiple ones if needed. For more granular data, visit your [Live Process Page][4].
```

Reply:
Should we also clarify here with - since with larger process counts numbers are calculated as approximate? @robertjli @mbotarro what's the best way to articulate this?

Reply:
few thousands processes -> few thousand processes

Reply:
> since with larger process counts numbers are calculated as approximate
I'm not sure this is true, frankly. I believe that we simply count the number of results that we get in our backend. Curious to hear what @robertjli and @mbotarro have to say.

Reply:
FYI - I addressed all the other points!